backend/package.json

```diff
@@ -6,7 +6,7 @@
     "scripts": {
         "start": "node index.js",
         "live": "nodemon index.js",
-        "test": "jest",
+        "test": "jest --maxWorkers=1",
```

Comment:
Usually, Jest will run the tests in parallel. If some tests uses a client to test out the event handlers, there will be socket port conflicts. Enforcing only one max worker might mean we lose some of the benefits of parallelisation but at least 
 by making the tests run sequentially, we can separate out the tests into different files.

Lmk if there is a better way of doing it please!